db/messages.go

```diff
@@ -24,11 +28,12 @@ func InsertMessage(SSID uint64, message []byte, timestamp string, sentOrReceived
 	if sentOrReceived != Sent && sentOrReceived != Received {
 		log.Fatalf("Invalid entry for sent/received - must be 0 or 1. Instead, received a %d", sentOrReceived)
 	}
+	encrypted_message := AESEncrypt(string(message), []byte(db.AESKey))
```

Comment:
Would prefer if this were camelcase tbh.

Reply:
I can make Encrypt camelcase, but not Decrypt since it is used in server. So either we keep both titlecase or we do one camelcase and one titlecase.

Reply:
Not AESEncrypt, I meant `encrypted_message`

Reply:
I definitely knew that...